SU2_CFD/include/solvers/CTransLMSolver.hpp

```diff
@@ -39,6 +39,72 @@
 class CTransLMSolver final : public CTurbSolver {
 private:
 
+  LM_ParsedOptions options;
+  TURB_FAMILY TurbFamily;
+
+  su2double ReThetaC_Correlations(const su2double Tu, const su2double Re_Theta_t){
+
+    su2double rethetac = 0.0;
```

Comment:
Don't forget to put these in a common place.

Reply:
Sure! Unfortunately, I am really busy at the moment, but next week I'll make all the changes and also update the test case repo.